apps/designer/app/canvas/shared/sync/use-sync.ts

```diff
@@ -17,7 +17,7 @@ export const useSync = ({ project }: { project: Project }) => {
         method: "post",
         body: JSON.stringify({
           transactions: entries,
-          treeId: project.devTreeId,
+          treeId: project.devBuild?.pages.homePage.treeId,
```

Comment:
this will depend on the currently selected page, right? What's the plan, is this temporary and needs a todo?

Reply:
Yeah, temporary. We wanted to only change schema in this PR, and make minimal changes to the reset of the code, just to make it work.

Will add todos.

Reply:
gotcha, as you wish, as long as we can merge it and potentially release it, all good